docs/managing-dragonfly/operator/authentication.md

```diff
@@ -0,0 +1,277 @@
+---
+sidebar_position: 2
+---
+
+# Dragonfly Instance Authentication
+
+This guide provides step-by-step instructions for setting up Dragonfly with authentication. Currently, Dragonfly supports two types of authentication:
+
+- [Password-based authentication](#password-based-authentication) through a secret
```

Comment:
What about ACL's? Also keep in mind that if:

`requirepass=mypass` && `ACL SETUSER default newpass`

Then `AUTH mypass` will fail while `AUTH newpass` will succeed

Reply:
But it feels like a Dragonfly docs thing right? and we don;t do anything w.r.t auth in the Operator yet. Worth to mention when we add something there? wdyt?